modules/data-entry/src/main/frontend/src/questionnaire/Question.jsx

```diff
@@ -129,8 +129,9 @@ function Question (props) {
             ( labels ?
               <List>
                 { labels.map( (item, idx) => {
+                  let value = defaultDisplayFormatter ? defaultDisplayFormatter(item, idx) : item;
                   return(
-                    <ListItem key={existingAnswer[0] + idx}> {defaultDisplayFormatter ? defaultDisplayFormatter(item, idx) : item}</ListItem>
+                    value ? <ListItem key={existingAnswer[0] + idx}> {value}</ListItem> : null
```

Comment:
If `value` is `0`, nothing gets displayed. This is not pretty, but catches this one case:
```suggestion
                    value || value === 0 ? <ListItem key={existingAnswer[0] + idx}> {value}</ListItem> : null
```